libbeat/processors/actions/decode_json_fields.go

```diff
@@ -17,18 +17,21 @@ type decodeJSONFields struct {
 	fields       []string
 	maxDepth     int
 	processArray bool
+	target       string
```

Comment:
Thanks for that tip 👍 Still have to learn a lot when coming from OO. There is one more question. Like you say if we want to add to root it now looks like
``` 
{
  "": {
    "test": {
      "test": "test"
    }
  },
  "@timestamp": "2016-12-14T08:20:16.157Z",
  "beat": {
    "hostname": "4201halwsd00001",
    "name": "4201halwsd00001",
    "version": "6.0.0-alpha1"
  },
  "input_type": "log",
  "message": "{\n    \"test\": \"test\",\n    \"test\": {\n        \"test\": \"test\"\n    }\n}",
  "offset": 73,
  "source": "input.json",
  "type": "log"
}
```
It looks right?